vslib/inc/MACsec.h

```diff
@@ -0,0 +1,160 @@
+#pragma once
+
+#include "TrafficFilter.h"
+#include "HostInterfaceInfo.h"
+
+#include <swss/selectableevent.h>
+#include <saitypes.h>
+
+#include <string>
+#include <thread>
+#include <cinttypes>
+
+namespace saivs
+{
+
+using macsec_sci_t = std::string;
+using macsec_an_t = std::uint16_t;
+using macsec_pn_t = std::uint64_t;
+
+struct MACsecAttr
+{
+    std::string m_veth_name;
```

Comment:
in this file you have 5 classes/structs, move each class/struct to separate file, 1 class per file